src/utils/StorageUtil.ts

```diff
@@ -0,0 +1,12 @@
+import AsyncStorage from '@react-native-async-storage/async-storage';
+
+export const setDeepLinkWallet = (universalLink: string) => {
```

Comment:
Should these be async? To ensure ts catches this and we don't run into unexpected bugs where we'd expect link to be set without awaiting.